frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -93,34 +93,37 @@ export const updateDashboardAndCards = createThunkAction(
           .map(async dc => CardApi.update(dc.card)),
       );
 
-      // update the dashboard cards and tabs
       const dashcardsToUpdate = dashboard.dashcards.filter(dc => !dc.isRemoved);
-      const updatedCardsAndTabs = await DashboardApi.updateCardsAndTabs({
-        dashId: dashboard.id,
-        cards: dashcardsToUpdate.map(dc => ({
-          id: dc.id,
-          card_id: dc.card_id,
-          dashboard_tab_id: dc.dashboard_tab_id,
-          action_id: dc.action_id,
-          row: dc.row,
-          col: dc.col,
-          size_x: dc.size_x,
-          size_y: dc.size_y,
-          series: dc.series,
-          visualization_settings: dc.visualization_settings,
-          parameter_mappings: dc.parameter_mappings,
-        })),
-        ordered_tabs: (dashboard.ordered_tabs ?? [])
-          .filter(tab => !tab.isRemoved)
-          .map(({ id, name }) => ({
-            id,
-            name,
+      const updateCardsAndTabs = () =>
```

Comment:
you can keep it as before, but await in Promise.all:

`const updatedCardsAndTabs = DashboardApi.updateCardsAndTabs({`

Reply:
Oh great

Reply:
Updated [Skip the anonymous function](https://github.com/metabase/metabase/pull/34796/commits/c58d8f0607966cf42f3bbe60761e99539f8e257a)